Generate changelog for 2.11.6

Totktonada opened the pull request:
Also, remove unreleased/ entries.

lenkis commented on changelogs/2.11.6.md:
```diff
@@ -0,0 +1,158 @@
+The reason is that the sorting of `double` fields would always compare the field
+values as C-language doubles, even when actual integers are stored. This works
+fine for small numbers, but starting from 2^53 the double-style comparison
+looses precision. For example, 18446744073709551615 and 18446744073709551614
```
```suggestion
loses precision. For example, 18446744073709551615 and 18446744073709551614
```

Totktonada replied:
Fixed.

Totktonada replied:
Sent for `3.4.0`, `3.3.2`, `3.2.2` in #11196.

lenkis commented on changelogs/2.11.6.md:
```diff
@@ -0,0 +1,158 @@
+Unfortunately, even if the user doesn't have the `double` type in any vinyl
+indexes, but **ever had it before**, it is still unsafe. A `double` index
+altered to `number` or `scalar` would remain broken. It is **very important that
+the users rebuild all the vinyl indexes that have or ever had the `double` field
```
```suggestion
users rebuild all the vinyl indexes that have or ever had the `double` field
```

Totktonada replied:
Fixed.

Totktonada replied:
Sent for `3.4.0`, `3.3.2`, `3.2.2` in #11196.

lenkis commented on changelogs/2.11.6.md:
```diff
@@ -0,0 +1,158 @@
+## Overview
+
+2.11.6 is the next stable release in the [long-term support (LTS)
+version][release_policy] 2.11.x release series.
```
```suggestion
version][release_policy] of the 2.11.x release series.
```

Totktonada replied:
Fixed.

Totktonada replied:
At the end, I rewrote the release notes template almost entirely, see #11210.

lenkis commented on changelogs/2.11.6.md:
```diff
@@ -0,0 +1,158 @@
+### Replication
+
+* RAFT requests are now sent with 0 (not local) GROUP_ID (gh-10727).
+* Fixed a bug when the Raft state wasn't sent during the `META_JOIN` stage,
```
```suggestion
* Fixed a bug when the RAFT state wasn't sent during the `META_JOIN` stage,
```

Totktonada replied:
Fixed.

Totktonada replied:
Sent for `3.2.2` in #11197. Can't send it for 3.3.2 and 3.4.0, because the change is already released in 3.3.0.

lenkis commented on changelogs/2.11.6.md:
```diff
@@ -0,0 +1,158 @@
+  The bug couldn't result in inconsistent query results, but it could lead to
+  performance degradation and increased disk usage (gh-10820, gh-10822).
+* Fixed a bug when joining a new replica to a master instance that experiences
+  a heavy write load would severely degrade the master instance performance.
```
```suggestion
  a heavy write load severely degrades the master instance performance.
```

Totktonada replied:
Fixed.

Totktonada replied:
Sent for `3.2.2` in #11197. Can't send it for 3.3.2 and 3.4.0, because the change is already released in 3.3.0.

Totktonada commented:
@lenkis Thanks! Updated according to the review comments.

I'll reflect these remarks in the 3.x branches and the release notes templates separately.

<details>
<summary>Diff</summary>

```diff
diff --git a/changelogs/2.11.6.md b/changelogs/2.11.6.md
index fd6eccdd81..38a91cc9b9 100644
--- a/changelogs/2.11.6.md
+++ b/changelogs/2.11.6.md
@@ -13,7 +13,7 @@ drop-in replaceable by the `number` field type.
 The reason is that the sorting of `double` fields would always compare the field
 values as C-language doubles, even when actual integers are stored. This works
 fine for small numbers, but starting from 2^53 the double-style comparison
-looses precision. For example, 18446744073709551615 and 18446744073709551614
+loses precision. For example, 18446744073709551615 and 18446744073709551614
 would be considered the same values.
 
 Besides, point-lookups in such index are going to be broken by design too -
@@ -23,7 +23,7 @@ other integers around this point.
 Unfortunately, even if the user doesn't have the `double` type in any vinyl
 indexes, but **ever had it before**, it is still unsafe. A `double` index
 altered to `number` or `scalar` would remain broken. It is **very important that
-the users rebuild all the vinyl indexes that have or ever had the `double` field
+users rebuild all the vinyl indexes that have or ever had the `double` field
 type in them**. What is worse, just an alter `double` -> `number` **won't
 work**. The old index must be dropped and a new one must be created. In any
 order.
@@ -35,7 +35,7 @@ See more details in gh-9965 and [gh-11017][gh-11017].
 ## Overview
 
 2.11.6 is the next stable release in the [long-term support (LTS)
-version][release_policy] 2.11.x release series.
+version][release_policy] of the 2.11.x release series.
 
 The label "stable" means there are 2.11.x-based applications running in
 production for quite a while without known crashes, incorrect results or
@@ -87,7 +87,7 @@ all the new features of the 2.x series.
   The bug couldn't result in inconsistent query results, but it could lead to
   performance degradation and increased disk usage (gh-10820, gh-10822).
 * Fixed a bug when joining a new replica to a master instance that experiences
-  a heavy write load would severely degrade the master instance performance.
+  a heavy write load severely degrades the master instance performance.
   The fix should also speed up long-running scan requests (gh-10846).
 * Fixed a bug when a tuple could disappear from a multikey index in case it
   replaced a tuple with duplicate multikey array entries created in the same
@@ -107,7 +107,7 @@ all the new features of the 2.x series.
 ### Replication
 
 * RAFT requests are now sent with 0 (not local) GROUP_ID (gh-10727).
-* Fixed a bug when the Raft state wasn't sent during the `META_JOIN` stage,
+* Fixed a bug when the RAFT state wasn't sent during the `META_JOIN` stage,
   which could lead to split-brain (gh-10089).
 * Fixed a bug when anonymous replicas could participate in elections or even
   be chosen as a leader. It is now forbidden to configure a replica so
```
</details>
